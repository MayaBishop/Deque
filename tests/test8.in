#testing empty size and print commands with a deque of 3 along a deque containing nothing
enqueue_front 1
enqueue_back 2
enqueue_front 3
print
empty
size
clear
size
empty
print
