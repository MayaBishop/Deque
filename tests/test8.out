success
success
success
3 1 2
2 1 3
failure
size is 3
success
size is 0
success
